permissions/README.md

```diff
@@ -0,0 +1,220 @@
+# Permissions
+
+**Note:** This is currently a [_beta_ feature](https://github.com/avenga/couper/blob/master/docs/BETA.md).
+
+Suppose you have an API with four endpoints:
+* `/a`
+* `/b/send`
+* `/b/copy`
+* `/c`
+
+and you want to use couper for access control. No problem, you can use the `jwt` access control (see example [JWT Access Control](../jwt-access-control/README.md))
+
+```hcl
+server {
+  api {
+    access_control = ["Token"]
+
+    endpoint "/**" {
+      proxy {
+        backend = "api"
+      }
+    }
+  }
+}
+
+definitions {
+  jwt "Token" {
+    signature_algorithm = "RS256"
+    key_file = "pub-key.pem"
+  }
+
+  backend "api" {
+    origin = "http://api:8080"
+  }
+}
+```
+
+But additionally, the API endpoints have certain permissions that have to be granted to the requester to access them:
+
+* requests to `/a` need permission "a"
+* requests to `/b/send` need permission "b:send"
+* requests to `/b/copy` need permission "b:copy"
+* `DELETE` requests to `/c` need permission "c:del"
+* `GET` requests to `/c` need no permission
+* other requests to `/c` need permission "c"
+
+You can configure this using the `beta_required_permission` attribute in the endpoint block.
+
+First, we replace the wildcard `endpoint` bock with three more specific `endpoint` blocks:
+
+
+```hcl
+server {
+  api {
+    access_control = ["Token"]
+
+    endpoint "/a" {
+      proxy {
+        backend = "api"
+      }
+    }
+
+    endpoint "/b/{action}" { # send, copy
+      proxy {
+        backend = "api"
+      }
+    }
+
+    endpoint "/c" {
+      proxy {
+        backend = "api"
+      }
+    }
+  }
+}
+```
+
+Then we set the required permission for `/a`:
+
+```hcl
+    endpoint "/a" {
+      beta_required_permission = "a"    # ←
+
+      proxy {
+        backend = "api"
+      }
+    }
+```
+That is simple. For the `/b/{action}` endpoint we can use a quoted template expression, because the permission consists of a prefix ("b:") and the value of the `action` path parameter ("send" or "copy").
```

Comment:
```suggestion
That is simple. For the `/b/{action}` endpoint we can use a quoted template expression (`${…}`), because the permission consists of a prefix ("b:") and the value of the `action` path parameter ("send" or "copy").
```